README.md

```diff
@@ -1,9 +1,1444 @@
+object with the attributes given below and for multiple local secondary indexes pass an array of
+JSON objects.(Can only create 5 LSI at max)
+
+-   Key name: **sort_key_name**
+
+    Description: Name of the local secondary sort key
+
+    Required: True
+
+    Input Type: String
+
+-   Key name: **sort_key_datatype**
+
+    Description: Datatype for the local secondary sort key(accepts only 3 types of values as
+    valid,that are **String, Number or Binary** )
+
+    Required: True
+
+    Input Type: String
+
+-   Key name: **attribute_projection**
+
+    Description: Type of attribute projection. There are the following types of projections
+
+    -   KEYS_ONLY: Only the index and primary keys are projected into the index.
+    -   INCLUDE: Only the specified table attributes are projected into the index. The list of
+        projected attributes is in NonKeyAttributes.
+    -   ALL: All of the table attributes are projected into the index.
+
+    Required: True
+
+    Input Type: String
+
+-   Key name: **non_key_attributes**
+
+    Description: A list of one or more non-key attribute names that are projected into the secondary
+    index. (Required only when the projection mode is set to INCLUDE)
+
+    Required: False
+
+    Input Type: Array of string values
+
+
+
+VALUE EXAMPLE
+
+-   Single JSON object for LSI
+
+                    {
+                      "sort_key_name": "lsi_one",
+                      "sort_key_datatype": "String",
+                      "attribute_projection": "INCLUDE",
+                      "non_key_attributes": [
+                          "one",
+                          "two",
+                          "three"
+                      ]
+                    }
+                
+
+-   Multiple JSON object for LSI
+
+                    [
+                      {
+                          "sort_key_name": "lsi_one",
+                          "sort_key_datatype": "String",
+                          "attribute_projection": "INCLUDE",
+                          "non_key_attributes": [
+                              "one",
+                              "two",
+                              "three"
+                          ]
+                      },
+                      {
+                          "sort_key_name": "lsi_two",
+                          "sort_key_datatype": "String",
+                          "attribute_projection": "ALL"
+                      }
+                    ]
+                
+
+
+
+
+
+
+
+**Global Secondary Index**
+
+To create a global secondary index(GSI), Partition key becomes necessary. Sort key is optional. The
+parameters for Global Secondary Indexes remain the same as Local Secondary Index, just Partition key
+name and Partition key name datatype are added and they become required parameters. Whereas Sort key
+name and Sort key datatype become optional. The provisional read and write capacity units are the
+same as the original table. (Can only create 20 GSI at max)
```

Comment:
```suggestion
A partition key is required to create a global secondary index(GSI) while a sort key is optional. All other
parameters for Global Secondary Indexes remain the same as those for Local Secondary Indexers; however, partition key
name and partition key name datatype are additional required parameters whereas sort key
name and sort key datatype are optional parameters. The provisional read and write capacity units are the
same as the original table (can only create 20 GSI at max).

Reply:
Updated the code and resolved the issue.